service/api/types.go

```diff
@@ -19,6 +19,11 @@ var ErrNotExecutable = proc.ErrNotExecutable
 type DebuggerState struct {
 	// Running is true if the process is running and no other information can be collected.
 	Running bool
+	// Recoring is true if the process is currently being recorded and no other
+	// information can be collected. While the debugger is in this state
+	// sending a Command("halt") request will halt the recording, every other
```

Comment:
halt should probably be changed to StopRecording?

Reply:
Done, thanks.